app/assets/javascripts/models/story.js

```diff
@@ -40,9 +40,17 @@ var Story = module.exports = Backbone.Model.extend({
     this.setReadonly();
   },
 
-  setReadonly: function() {
-    if(this.get('state') === 'accepted' && this.get('accepted_at') !== undefined)
-      this.isReadonly = true;
+  setReadonly: function () {
+    var accepted = this.get('state') === 'accepted' && this.get('accepted_at') !== undefined;
+    var isGuest = (
+          this.collection !== undefined &&
```

Comment:
Align this like you did in the file bellow.

Reply:
Long lines are not problem?

Reply:
I mean, like here https://github.com/Codeminer42/cm42-central/pull/243/files#diff-5d8aecce3a76454de0b472c96fa1b6a8R346

Reply:
Done